simple-overview.md

```diff
@@ -1,19 +1,27 @@
 # What is TACo?
 
-TACo is a plug-in service that enables the safe sharing of private or sensitive data within Web3 applications. Private data is encrypted by a data owner and remains encrypted until it reaches the device of a recipient. From a privacy point of view, it is comparable to the end-to-end encryption that a mobile messaging service like Signal offers.&#x20;
+TACo is a plug-in service that enables the safe sharing of private or sensitive data within Web3 applications.
+Private data is encrypted by a data producer and remains encrypted until it reaches the device of a recipient.
```

Comment:
As commented in #2  - we should also investigate gitbook settings since subsequent edits made from the UI may re-introduce the removed characters / remove the modified line breaks in this PR.